bin/update_pods.py

```diff
@@ -377,6 +398,11 @@ def main():
 
     global SRC_ROOT
     SRC_ROOT = args.src_root
+
+    if not SRC_ROOT:
+        SRC_ROOT = os.getcwd()
```

Comment:
Might want to print a warning here that src_root was not passed in and we are falling back to cwd

Reply:
@rahul-malik this is a good idea - we should deff print the `SRC_ROOT` somewhere in output.

Reply:
It looks like we already print this out:
```
405     print("Updating pods in " + SRC_ROOT)
```